libbeat/cmd/instance/beat.go

```diff
@@ -504,10 +504,6 @@ func (b *Beat) launch(settings Settings, bt beat.Creator) error {
 
 	logp.Info("%s start running.", b.Info.Beat)
 
-	// Launch config manager
-	b.Manager.Start(beater.Stop)
```

Comment:
Are there any dependencies on this outside this repository that would break? Custom beats? Maybe the new cloud beat?

Reply:
That would be only APM-Server and Cloudbeat, I have PR to fix in APM.